packages/expo-file-system/android/src/main/java/expo/modules/filesystem/next/FileSystemNextModule.kt

```diff
@@ -98,11 +99,17 @@ class FileSystemNextModule : Module() {
       }
 
       Property("md5") { file ->
-        file.md5
+        try {
+          file.md5
+        } catch (e: FileNotFoundException) {
+          null
```

Comment:
Don't you need to change the types as well?

Reply:
No, it's `string | null` already